bot/advisors.js

```diff
@@ -157,12 +172,34 @@ class Advisors {
     }
 
     poe(data) {
-
+      let id = _.get(data, 'prisonofelders.display.activityHash', '');
+      let message = 'Prison of Elders is: ';
+      return this.fetchActivity(id).then(
+          (rawactivity) => {
+              let activity = _.get(rawactivity, 'Response.data.activity', {});
+              message += '*'+_.get(activity, 'activityName', 'Unknown')+'*'+'\n\n';
+              return message;
+          }
+      );
     }
 
     coe(data) {
-
+      let id = _.get(data, 'elderchallenge.display.activityHash', '');
+      return this.fetchActivity(id).then(
+          (rawactivity) => {
+              let activity = _.get(rawactivity, 'Response.data.activity', {});
+              let attachments = _.concat(
+                  [
+                    {'title': 'Challenge of the Elders'}
```

Comment:
attachments should have a fallback key w/ a plain text version (just copypasta).. i think this is why xur keeps yelling at me and then it goes away